src/views/TrackPositionAnimation/TrackPositionAnimationView.tsx

```diff
@@ -171,8 +171,15 @@ const TrackPositionAnimationView: FunctionComponent<TrackPositionAnimationProps>
 
     const { focusTime, setTimeFocus } = useTimeFocus()  // state imported from recording context
     const findNearestTime = useTimeLookupFn(animationState)
-    useEffect(() => matchFrameToFocus(focusTime, findNearestTime, animationState, animationStateDispatch), [focusTime])
-    useEffect(() => matchFocusToFrame(animationState, focusTime, setTimeFocus), [animationState.isPlaying])
+    const animationCurrentTime = animationState?.frameData[animationState?.currentFrameIndex]?.timestamp
+    useEffect(() => {
+        matchFrameToFocus(focusTime, findNearestTime, animationStateDispatch)
+    }, [focusTime, findNearestTime])
```

Comment:
I'm not sure that running this whenever `findNearestTime` updates is exactly right--but in practice it doesn't matter since that function should only update when we change the list of underlying frames, which in current usage never actually happens.

Reply:
If we want to be *really* picky about it, we could do 
```javascript
const frameUpdater = useCallback((focusTime: number | undefined) => {
    matchFrameToFocus(focusTime, findNearestTime, animationStateDispatch)
}, [findNearestTime, animationStateDispatch])
useEffect(() => frameUpdater(focusTime), [focusTime])
```
but that is probably overkill here.